src/biogeochem/CNCStateUpdate1Mod.F90

```diff
@@ -274,10 +274,8 @@ subroutine CStateUpdate1( num_soilc, filter_soilc, num_soilp, filter_soilp, &
 
          ! phenology: litterfall fluxes
          cs_veg%leafc_patch(p) = cs_veg%leafc_patch(p) - cf_veg%leafc_to_litter_patch(p)*dt
+         cs_veg%leafc_patch(p) = cs_veg%leafc_patch(p) - cf_veg%leafc_to_biofuelc_patch(p)*dt
```

Comment:
This flux is not set to 0 if running with use_crop false, so it isn't safe to reference it here. I feel the best thing to do is to move this line down into the `if (ivt(p) >= npcropmin)` conditional block below (near the reference to `livestemc_to_biofuelc`).